src/config.ts

```diff
@@ -0,0 +1,9 @@
+import GenericConfig from './platform/generic';
+import VercelConfig from './platform/vercel';
+import NetlifyConfig from './platform/netlify';
+
+export const isVercel = process.env.AXIOM_INGEST_ENDPOINT || process.env.NEXT_PUBLIC_AXIOM_INGEST_ENDPOINT;
+export const isNetlify = process.env.NETLIFY == 'true';
+
+const config = isVercel ? new VercelConfig() : isNetlify ? new NetlifyConfig() : new GenericConfig();
```

Comment:
Can we make this if/else statements for readability?

Reply:
done, will test it again now.

Reply:
works like a charm on vercel and netlify 👍